otel-extensions/src/main/java/org/hypertrace/agent/otel/extensions/HypertraceResourceProvider.java

```diff
@@ -40,6 +49,25 @@ public Resource createResource(ConfigProperties config) {
       builder.put(ResourceAttributes.CONTAINER_ID, containerId);
     }
     builder.put(ResourceAttributes.SERVICE_NAME, agentConfig.getServiceName().getValue());
+    builder.put(ResourceAttributes.TELEMETRY_SDK_NAME, "hypertrace");
+    builder.put(ResourceAttributes.TELEMETRY_SDK_LANGUAGE, "java");
+    String agentVersion = getAgentVersion();
+    builder.put(ResourceAttributes.TELEMETRY_SDK_VERSION, agentVersion);
+    builder.put(ResourceAttributes.TELEMETRY_AUTO_VERSION, agentVersion);
+    builder.put(AttributeKey.stringKey(HYPERTRACE_MODULE_NAME), "java");
+    builder.put(AttributeKey.stringKey(HYPERTRACE_MODULE_VERSION), agentVersion);
```

Comment:
@davexroth related to the wiki doc - it seems there is quite large redundancy in the version attributes. 

Could we use only a single one if the values are repeated?

Reply:
I replied to your comment on the wiki.